app/controllers/api/users_controller.rb

```diff
@@ -42,7 +45,7 @@ def update
     def user_params
       params.require(:data).permit :username,
                                    :age,
-                                   :tags
+                                   :tags => []
```

Comment:
Does this work? Never seen this syntax before.

You can also use `Array(tags)` to cast it to an array if it's not one already.